src/components/common/Balance.js

```diff
@@ -64,7 +64,7 @@ const Balance = ({ amount, symbol, className }) => {
             {amount && symbol !== false && symbol !== 'near' && <span className='symbol'>Ⓝ</span>}
             {amount
                 ? amountShow
-                : <div className="dots">Loading</div>
+                : <div className="dots">Retrieving balance</div>
```

Comment:
@marcinbodnar can we please make this a translate string too

Reply:
@corwinharrell It's not a good idea to change this, this component is used to show all amounts in Near in Wallet - if you don't like `loading` we can change it but it should be more universal and also shorter than `Retrieving balance`.

Reply:
@marcinbodnar I see. My assumption was that due to the fact that this was called "Balance", that it was scoped to just balances and not all amounts. I thought that saying "retrieving balances..." was more specific and insightful. If this component goes beyond balances and is used for other amounts as well, perhaps we can change its name to reflect its more general usage?

Reply:
@corwinharrell true, it does not reflect its usage anymore. I will add a translation for now, would you like me to change `Loading` for something different?

Reply:
@marcinbodnar if this component is more general-purpose than just balances I think we can leave it as "Loading..." Eventually I'd like to take a pass at some of our loading states and introduce some nicer loaders etc...